src/core/src/bootstrap/ConfigurationFactory.py

```diff
@@ -20,7 +20,7 @@
 from core.src.bootstrap.Constants import Constants
 from core.src.bootstrap.EnvLayer import EnvLayer
 
-from core.src.core_logic.ConfigurePatching import ConfigurePatching
+from core.src.core_logic.ConfigurePatchProcessor import ConfigurePatchProcessor
```

Comment:
Consider ConfigurePatchManager since you have RebootManager

Reply:
ConfigurePatchingProcessor

Reply:
Done